assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/TableView.tsx

```diff
@@ -7,29 +7,23 @@ import datesListTableHeader from './datesListTableHeader';
 import datesListTableRow from './datesListTableRow';
 import useDatetimeItem from '@edtrServices/apollo/queries/datetimes/useDatetimeItem';
 
-import './editor-date-entities-list-view.css';
+import { TableViewProps } from './types';
+
+import './styles.scss';
 
 const noZebraStripe = ['row', 'stripe', 'name', 'actions'];
 
 /**
  * EditorDateEntitiesListView
  * Displays event date details in a standard list table like view
  */
-const TableView = ({
-	className,
-	displayDates,
-	entities: datetimes,
-	allEventDates,
-	showDate,
-	setEntityIds,
-	setSortBy,
-	...props
-}) => {
+const TableView: React.FC<TableViewProps> = ({ className, displayDates, entities: datetimes, showDate, ...props }) => {
```

Comment:
May be we should rename `displayDates` and `showDate` to some more explicit and clear names, that can be done in another PR.

Reply:
agree with this suggestion :+1: